### DISCUSSION TITLE
Im trying to change my entity variant

### DISCUSSION THREAD

**Discussion by jason7141** [bridge-core/editor#914] [Q&A]
I'm trying to change my entity variant using interact but it's not changing the texture any idea why? This is how it's setup
![Screenshot_20230509-234946_Chrome](https://github.com/bridge-core/editor/assets/105606159/1abe24d7-bcba-43a4-a986-6328f80d5db6)
![Screenshot_20230509-234928_Chrome](https://github.com/bridge-core/editor/assets/105606159/00b514a1-f8c1-49dc-a96f-6d772216947e)
![Screenshot_20230509-234908_Chrome](https://github.com/bridge-core/editor/assets/105606159/32363a5d-9444-4144-8650-7ae58f8b952f)

**Comment by ewanpeebles** [comment]
I think what you're looking for is a [render controller.](https://wiki.bedrock.dev/entities/render-controllers.html). Hope I'm not too late for this!
